psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -687,6 +691,26 @@ def scf_print_preiterations(self):
     )
     core.print_out("   -------------------------------------------------------\n\n")
 
+def scf_print_preiterations_small(self):
```

Comment:
Would this be cleaner as part of the `scf_print_preiterations` function with `small=True`?

Reply:
Yes, i think so. I wasn't sure if I need to make changes somewhere else if I modify the function and chose the saver approach for now.